src/components/common/Chat/index.js

```diff
@@ -0,0 +1,66 @@
+import useTranslation from 'hooks/useTranslation';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectAuth } from 'services/auth/auth';
+import { useGetConversationsQuery } from 'services/model/conversations';
+import { DataScroller } from 'primereact/datascroller';
+import './styles.scss';
+import { error } from 'react-toastify-redux';
+import { useCallback } from 'react';
+
+const Chat = () => {
+  const t = useTranslation();
+  const { user } = useSelector(selectAuth);
+  const { data: conversations } = useGetConversationsQuery(user.id);
+  const matches = conversations?.matches || [];
+  const dispatch = useDispatch();
+
+  const onClickItem = useCallback(() => {
+    dispatch(error(t('global.notImplemented')));
+  }, [dispatch, t]);
+
+  const itemTemplate = match => {
+    return (
+      <>
+        <div
+          className="chat__item w-full"
+          onClick={onClickItem}
+          onKeyDown={onClickItem}
+          role="button"
+          tabIndex={0}
+        >
+          <img
+            className="chat__image"
+            src={match.user.avatar.small_thumb_url || './default-profile.png'}
```

Comment:
It's more common leave the paths/urls to the imports section and only use names/aliases in the jsx.

Reply:
Right! will fix